plugin/core/open.py

```diff
@@ -15,6 +15,21 @@
 opening_files = {}  # type: Dict[str, Tuple[Promise[Optional[sublime.View]], ResolveFunc[Optional[sublime.View]]]]
 
 
+def _return_existing_view(flags: int, existing_view_group: int, active_group: int, specified_group: int) -> bool:
```

Comment:
Since we don't need `existing_view_group` and `active_group` to be passed here separately, we can just combine those into a single `is_view_in_active_group`.

Reply:
Turns out we needed `existing_view_group` and `active_group` to be passed here separately.

See: https://github.com/sublimelsp/LSP/pull/2031#discussion_r958132385